### PR TITLE
Add blog post on testing Camel K with YAKS

### PR DESCRIPTION
**Pull request by christophd**

### CODE REVIEW

**Review by squakez — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

**Review by davsclaus — APPROVED** [review]

**Comment by davsclaus** [comment]
@christophd 

```
[check:html     ] public/blog/2023/01/camel-k-yaks/index.html
[check:html     ]    32:8[9](https://github.com/apache/camel-website/actions/runs/4025277259/jobs/6952555920#step:5:10)5  error  For links within camel.apache.org use relative links, found: https://camel.apache.org/blog/2022/11/camel-k-jbang/  camel/relative-links
[check:html     ]    60:113  error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]    62:22   error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]    65:44   error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]    68:7    error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]    70:7    error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]    72:7    error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]    95:7    error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]    97:7    error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]   [10](https://github.com/apache/camel-website/actions/runs/4025277259/jobs/6952555920#step:5:11)1:7    error  Trailing whitespace                                                                                                no-trailing-whitespace
[check:html     ]   105:7    error  Trailing whitespace                                                                                                no-trailing-whitespace
```

**Comment by christophd** [comment]
@davsclaus thanks! I will have a look

**Comment by christophd** [comment]
The `Trailing whitespace` checks really give me some headache. I failed to build the website locally (on MacOS) so I am not able to really see the generated HTML sources to check for the trailing whitespaces. I made the CI job to always provide the generated HTML source as artifacts (also on failed CI jobs) so I can have a look there.

At the moment I have no idea where the trailing whitespaces may come from

File: content/blog/2023/01/camel-k-yaks/index.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+Just write the test file and run with:
+
+```shell script
+$ yaks run fruit-service.feature --local
+```
+
+You should see some log output like this:
+
+```
+INFO |
```
Comment (zregvart) [inline content/blog/2023/01/camel-k-yaks/index.md:168]: @christophd try removing trailing whitespace like this:

```suggestion
INFO |
```

Comment (christophd) [reply content/blog/2023/01/camel-k-yaks/index.md:168]: ah! my IDE does not touch these pre-formatted code blocks. thx @zregvart

**Comment by oscerd** [comment]
+1

**Comment by christophd** [comment]
@zregvart I can't see the post being published on the website and CI jobs do have failures in Jenkins https://ci-builds.apache.org/blue/organizations/jenkins/Camel%2FCamel.website/detail/main/1125/pipeline

**Comment by zregvart** [comment]
> @zregvart I can't see the post being published on the website and CI jobs do have failures in Jenkins [ci-builds.apache.org/blue/organizations/jenkins/Camel%2FCamel.website/detail/main/1125/pipeline](https://ci-builds.apache.org/blue/organizations/jenkins/Camel%2FCamel.website/detail/main/1125/pipeline)

@christophd I've canceled a hung build and triggered the build manually. The [blog post](https://camel.apache.org/blog/2023/01/camel-k-yaks/) is published now.

**Pull request merged**